src/components/ComponentExample/component-example.scss

```diff
@@ -43,7 +43,7 @@
   bottom: 0;
   color: $text-02;
   @include carbon--type-style('caption-01');
-  padding: 0.5rem 1rem;
+  padding: $spacing-03 1rem;
```

Comment:
did you want to update the `1rem` to also use a spacing token?

Reply:
good catch 👍